pulp_plugin_template/app/models.py

```diff
@@ -0,0 +1,55 @@
+from gettext import gettext as _
+from logging import getLogger
+
+from pulpcore.plugin.models import (Artifact, Content, ContentArtifact, RemoteArtifact, Importer,
+                                    ProgressBar, Publisher, RepositoryContent, PublishedArtifact,
+                                    PublishedMetadata)
+from pulpcore.plugin.tasking import Task
+
+log = getLogger(__name__)
+
+
+class PluginTemplateContent(Content):
+    """
+    The "plugin-template" content type.
+    """
+    TYPE = 'plugin-template'
+
+    # path = models.TextField(blank=False, null=False)
+    # digest = models.TextField(blank=False, null=False)
+    # objects = 
+
+    #class Meta:
+    #    unique_together = ()
+
+    @classmethod
+    def natural_key_fields(cls):
```

Comment:
This seems dangerous since _unique_together_ can be either: ("name", "version")` or `(("name", "version"), ..)`

Reply:
Do you suggest to complete this task first? https://pulp.plan.io/issues/3025

Reply:
I'm not sure it needs to be done first but suggest you remove this impl here for this PR.

Reply:
No, 3025 has a checklist item to vet on pulp-dev.  Suggest you just leave this in for now so you are not blocked.